packages/lesswrong/lib/routes.ts

```diff
@@ -583,7 +587,9 @@ const eaLwAfForumSpecificRoutes = forumSelect<Route[]>({
       componentName: 'EAHome',
       description: "Research, discussion, and updates on the world's most pressing problems. Including global health and development, animal welfare, AI safety, and biosecurity.",
       enableResourcePrefetch: true,
-      sunshineSidebar: true
+      sunshineSidebar: true,
+      hasLeftNavigationColumn: true,
+      navigationFooterBar: true,
```

Comment:
Since we don't use `navigationFooterBar` it's probably clearer to exclude it from the EAF routes